Week2/349_Intersection_of_2_arrays.cpp

```diff
@@ -0,0 +1,47 @@
+class Solution {
+public:
+    vector<int> intersection(vector<int>& nums1, vector<int>& nums2) {
+        //if (nums1.size() > nums2.size()) return intersection(nums2, nums1);
+
+	// The idea here is to use some hash table data structure (set or map).
```

Comment:
I also done this with unordered set first.
But with unordered set running time was slightly bigger I guess it is because of bigger hidden constant of operation erase in hset than operator [] in hmap.
https://leetcode.com/submissions/detail/80045924/